backend/utils/request/request.py

```diff
@@ -43,7 +44,7 @@ def make_http_request(
         return return_val
     except RequestException as e:
         logger.error(f"HTTP request error: {e}")
-        raise e
+        raise APIError(f"Error occured while invoking POST API Variable : {str(e)}")
```

Comment:
@harini-venkataraman I hope this gets propagated to the FE also as expected, my concern was around
1. User makes a syntax error, how do we convey this to the user?
1. User's POST call itself errors out

Some pointers for later
1. The final message shown to the user should contain context on the prompt key, document used, URL invoked and the actual error

Reply:
@chandrasekharan-zipstack The syntax check for variables is not handled yet. Will be picking it up in next version, and will add the details in error message.